File: apps/s-2-admin/src/api/attendee/AttendeeCreateInput.ts
import { BookingWhereUniqueInput } from "../booking/BookingWhereUniqueInput";

export type AttendeeCreateInput = {
  email: string;
  name: string;
  timeZone: string;
  locale?: string | null;
  booking?: BookingWhereUniqueInput | null;
};

File: apps/cactus-demo/src/APIs/Attendee/Dtos/AttendeeCreateInput.cs
namespace CactusDemo.APIs.Dtos;

public class AttendeeCreateInput
{
    public int? Id { get; set; }

    public string Email { get; set; }

    public string Name { get; set; }

    public string TimeZone { get; set; }

    public string? Locale { get; set; }

    public Booking? Booking { get; set; }
}
